geniza/corpus/views.py

```diff
@@ -69,9 +67,11 @@ def get_queryset(self):
                     .highlight("description", snippets=3, method="unified")
                     .also("score")
                 )  # include relevance score in results
-
-            # sorting TODO; for now, order by relevance
-            documents = documents.order_by("-score")
+            if search_opts["sort"]:
+                documents = documents.order_by(self.solr_sort[search_opts["sort"]])
+            else:
+                initial_sort = self.initial["sort"]
+                documents = documents.order_by(self.solr_sort[initial_sort])
```

Comment:
Initial sort needs to be set in `get_form_kwargs` so that it is always included in the search opts from the form. I guess right now it is doing the right thing but displaying the wrong value in the sort select in the rendered form.